Samples/BlazorSampleApp/MandelbrotExplorer/MandelbrotExtensions.cs

```diff
@@ -61,16 +64,181 @@ public static class MandelbrotExtensions
         }
 
 
+
         /// <summary>
         /// Calculate the Mandelbrot set single threaded on the CPU.
         /// </summary>
         /// <param name="buffer"></param>
         /// <param name="width"></param>
         /// <param name="height"></param>
         /// <param name="max_iterations"></param>
-        public static void CalcCPUSingle(int[] buffer, int[] display, float[] view, int max_iterations)
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
Based on our latest discussions, I think we should remove these functions to avoid people trying to use newly added types for general purpose compute (see below).